packages/runtimes/js/src/JSRuntime.js

```diff
@@ -74,14 +57,15 @@ export default new Runtime({
         )}]`;
       });
 
-      // $FlowFixMe
-      await bundle.assetGraph.addRuntimeAsset(bundleGroup, {
+      assets.push({
         filePath: __filename,
-        env: bundle.env,
         code: `module.exports = require('./bundle-loader')([${loaderModules.join(
           ', '
-        )}, ${JSON.stringify(bundleGroup.value.entryAssetId)}]);`
+        )}, ${JSON.stringify(bundleGroup.entryAssetId)}]);`,
+        bundleGroup
       });
     }
+
+    return assets;
```

Comment:
moved runtime plugins to a declarative api, rather than imperatively calling `addRuntimeAsset`.